### PR TITLE
Remove the admin/ prefix on Admin APIs for consistency

### PR DESCRIPTION
**Pull request by ThisIsMissEm**
Just noticed this whilst looking something up in the documentation, several of the admin API methods had `admin/` prefixed to them when the other sub-navigation items do not:

Screenshot 2023-05-31 at 2 51 10 am

### CODE REVIEW

**Comment by trwnh** [comment]
the other ones don't have prefixes because... they don't have prefixes. the sidebar indentation is purely visual

**Comment by ThisIsMissEm** [comment]
> the prefix is there to differentiate from the user-level equivalents. the ones without prefixes do not have a user-level equivalent.

I would argue that we should be consistent here. Even if you mistook them for user level instead of admin level, you'd see on their pages quiet clearly that you'd clicked on admin documentation and not user documentation

**Comment by vmstan** [comment]
> the prefix is there to differentiate from the user-level equivalents. the ones without prefixes do not have a user-level equivalent.

But things like `ip_blocks` and `email_domain_blocks` don't have user-level equivalents.

**Review by vmstan — APPROVED** [review]

**Comment by trwnh** [comment]
> But things like ip_blocks and email_domain_blocks don't have user-level equivalents.

yes, that's what i said. admin/accounts is prefixed because there's already an accounts entry, and so on.

**Comment by andypiper** [comment]
I appreciate all sides of the discussion here and it's a tricky one! I guess there are three approaches:

- as-is: some of the methods are prefixed, to attempt to disambiguate from similar `accounts` entries
- prefix all of the methods with `admin/` (which seems redundant as they are under the `admin` entry in the tree
- prefix none of the methods (as they are already under the `admin` tree entry, but may risk ambiguity with the methods of the same name under the `accounts` tree).

Bonus points for the all-prefixed/no-prefixed options is that the list of methods then ends up in an alphabetical sequence.

Of the three options, the one I personally prefer is the last one (i.e. this PR), because it is consistent across all of the other API docs. We *could* then add a small note to each of the affected pages to state that there is a similarly-named `accounts` entry, to suggest to the reader that they are in the right spot in the docs / API, but I'm not sure how helpful that would be.

One more point of view here is that, were we to have a properly-specced API with an OpenAPI definition (for example!), and to generate the documentation from that, then it would not add additional prefixes to methods on a partial basis.

In summary, as a tie-breaker, my choice here is to accept the PR, whilst understanding that there is a (IMO, small) chance of confusion between a handful of similarly-named methods at different locations in the URI structure.

**Review by andypiper — APPROVED** [review]
lgtm and tested as working locally. Thanks.

**Comment by trwnh** [comment]
so there's one other thing that i should mention as well -- being indented under a method doesn't actually imply anything about the path. the indentation is actually for grouping related methods together. for example, none of the entries in the sidebar under `accounts` are prefixed with `accounts/` because that's not their path.

**Comment by ThisIsMissEm** [comment]
> so there's one other thing that i should mention as well -- being indented under a method doesn't actually imply anything about the path. the indentation is actually for grouping related methods together. for example, none of the entries in the sidebar under `accounts` are prefixed with `accounts/` because that's not their path.

That's fair @trwnh, but this only affects the menu, not the actual contents of those pages, and as the menu is essentially nested lists this should be fine, I think? We're essentially saying "here's accounts in the context of admin" or "accounts in general"
